concepts/segment.md

```diff
@@ -0,0 +1,18 @@
+---
+grand_parent: Resources
+parent: Concepts
+title: Segment
+description: A sequence of characters
+---
+
+A segment is a unit that is part of a document or other file that is processed by a Computer-Aided Translation tool (CAT).
+Segments paired into source-target units are building blocks of translation memory. 
+Splitting text into such units allows to assign matches from translation memory and to identify repetitions.
+
+Segment content depends on the type of translated content:
+- In a document, a segment is usually a sentence or equivalent.
+- In a spreadsheet, a segment is a cell.
+- In user interface resource files, a segment can be a menu option, a text on a button, a field label, an error message text, etc.
+
+Segment is a [string](string.md).  
+Segment can be plain text or can include [tags or placeholders](../applications/advanced-concepts/tags-and-placeholders.md).
```

Comment:
Thanks for all your work and effort, @martab0!

I'm still not sure if adding this info to the `string` article or having a separate article is preferable.

Reply:
I tend to think that we need definitions of "string" and of "segment". Certainly I would rather avoid "line" when referring to how MT processes text. 
Thank you for the review, @cefoo!!!

Reply:
I agree that both are needed, but since they refer to the same concept, perhaps it would be best to add this information in the `string.md` article that already exists.

Perhaps we can move the first sentence of the last paragraph to the beginning, and make the transition from `string` to `segment` by saying

```
In the translation and localization field, a string is called a **segment**.
```

What do you think?

Reply:
I like the idea to combine "string" and "segment" into one, I am only wondering which one is more meaningful as a concept title? :) Perhaps call it "String and segment"???

Reply:
Shall we use the `string.md` file, and make `segment.md` redirect to it, [Wikipedia style](https://en.wikipedia.org/w/index.php?title=Czechia&redirect=no)?

Reply:
I think that `string` and `segment` have different senses and are not identical. The way I understand `segment` is a word, phrase or at most a paragraph. However, the whole text of a book would not be a segment.

However, all of these examples are strings (sequences of characters).